google/datalab/contrib/bigquery/commands/_bigquery.py

```diff
@@ -21,6 +20,10 @@ def _create_pipeline_subparser(parser):
                                                   'transform data using BigQuery.')
   pipeline_parser.add_argument('-n', '--name', type=str, help='BigQuery pipeline name',
                                required=True)
+  pipeline_parser.add_argument('-e', '--environment', type=str,
+                               help='The name of the Composer or Airflow environment.')
```

Comment:
It's only the name of the Composer right? Not used in Airflow?

Reply:
You're right. At the of putting the comment, I made it slightly future looking since we plan to work on an Airflow environment. Will change in a different PR mostly because I'm trying to get this in before demo time today.